README.md

```diff
@@ -2,6 +2,24 @@
 
 The Tax Foundation’s publication [Capital Cost Recovery across the OECD](https://taxfoundation.org/publications/capital-cost-recovery-across-the-oecd/) shows how capital allowances compare across OECD countries and how they have developed since 2000. The dataset we compiled for the years 1979 to 2020 is made available as a resource for research.
 
+## Latest Release
+The current version of this project is 1.1.0 (2021-03-31).
```

Comment:
GitHub actually has a feature for creating and tracking releases—try it [here](https://github.com/TaxFoundation/capital-cost-recovery/releases/new)

Reply:
That's awesome -- just added it. Thanks!